src/bytes_mut.rs

```diff
@@ -1476,6 +1530,7 @@ impl PartialEq<Bytes> for BytesMut {
     }
 }
 
+#[inline]
```

Comment:
I'm curious to know whether or not adding inline hints to these internal functions has a noticeable result on performance — in general, I expect `#[inline]` (not `#[inline(always)]`) to only be meaningful for `pub` functions, since it enables cross-crate inlining, and since it's a hint, `rustc` is pretty free to ignore it...